File: PyBiblioteca/socio.py
from libro import Libro

class Socio:
    def __init__(self, nombre: str, apellido: str, identificador: int):
        self.nombre = nombre
        self.apellido = apellido
        self.identificador = identificador

    def solicitar_prestamo(self, fecha_prestamo: str):
        print(f"Prestamo solicitado por {self.nombre} {self.apellido}, id: {self.identificador}")

    def devolver_prestamo(self):
        print(f"Devolucion de prestamo por {self.nombre} {self.apellido}, id: {self.identificador}")

    def informacion(self):
        print(f"nombre: {self.nombre}, apellido: {self.apellido}, numero de socio: {self.identificador}")

File: BibliotecaPackageKT/socio.kt
package Biblioteca
class Socio(val nombre: String, val apellido: String, val numeroSocio: Int) {

    fun solicitarPrestamo(libro: Libro, fechaPrestamo: String) {
        println("Préstamo solicitado por $nombre $apellido, Número de Socio: $numeroSocio")
        val prestamo = Prestamo(libro, this)
        prestamo.registrarPrestamo(fechaPrestamo)
    }

    fun devolverPrestamo(libro: Libro) {
        println("Devolución de préstamo por $nombre $apellido, Número de Socio: $numeroSocio")
        val prestamo = Prestamo(libro, this)
        prestamo.devolverPrestamo()
    }

    fun informacion() {
        println("Nombre: $nombre, Apellido: $apellido, Número de Socio: $numeroSocio")
    }
}
